### ISSUE TITLE
No configure script in tarball

### ISSUE THREAD

**Issue by mrobbert** [mej/nhc#48]
I'm installing on Ubuntu so I tried to follow the instructions to install from source which just say to download the latest tarball and run configure. That would be fine, but there is no configure in the tarball. There is a configure.in and I was able to generate the configure script with autoconf, but either the documentation needs to be updated or the tarball needs to be created after generating the configure script.

**Comment by mej** [comment]
Sounds like maybe you had GitHub generate a tarball from the Git repository rather than downloading the actual distribution tarball from the Releases page.  The distribution tarball is generated by running `make dist` from within the Git repo, and it *does* contain the `configure` script.  The Git repo itself does not (nor should it).

Try downloading `lbnl-1.4.2.tar.xz from [the 1.4.2 Release page](https://github.com/mej/nhc/releases/tag/1.4.2).  Make sure to AVOID the "Source code" links generated by GitHub; those don't work correctly for GNU autotools-based software.

**Comment by mrobbert** [comment]
OK, that makes sense, but I just went back to retrace my steps to figure out how I missed that and I found that the README.md has the wrong link. The link for "source tarball for the latest release" in the Installation section points to a tarball without the configure script, but then tells you to run ./configure. If you change that link to point to the correct file then the documentation will make more sense.
